src/backend/utils/html/replace-entities.js

```diff
@@ -0,0 +1,21 @@
+const entities = require('entities');
+
+function decode(codeElement) {
+  // decode twice for double encoded entities
+  const result = entities.decodeHTML(codeElement.innerHTML);
+  return entities.decodeHTML(result);
+}
+
+module.exports = function (dom) {
+  if (!(dom && dom.window && dom.window.document)) {
+    return null;
+  }
+
+  let fixedCodeElement = {};
+
+  dom.window.document.querySelectorAll('code').forEach((code) => {
+    fixedCodeElement = decode(code);
+    code.innerHTML = fixedCodeElement;
```

Comment:
The reason I put the `fixedCodeElement` there was so that I could return that value back to the tester 

https://github.com/Seneca-CDOT/telescope/blob/9f5747786f85a8cc7525766fbeecece1c520cab9/src/backend/utils/html/replace-entities.js#L20

Reply:
How about:
```js
code.innerHTML = decode(code);
return code.innerHTML;
```

Reply:
I mean in your tests, the same way that [syntax-highlight does it](https://github.com/Seneca-CDOT/telescope/blob/0764c770d57641e2bd866fb80e9beaa3253873d6/test/syntax-highlight.test.js#L8).

Reply:
> How about:
> 
> ```js
> code.innerHTML = decode(code);
> return code.innerHTML;
> ```
If we return the innerHTML of the element to the tests, I'm pretty sure that the decoded text that gets returned will become encoded again because its HTML.

Reply:
Sorry, I meant in your test. You could avoid returning here in your code, and add a return in your test.
Check out how `syntax-highlight` does it:
[Here](https://github.com/Seneca-CDOT/telescope/blob/0764c770d57641e2bd866fb80e9beaa3253873d6/src/backend/utils/html/syntax-highlight.js#L38) is the tool (it doesn't return anything)
And [here](https://github.com/Seneca-CDOT/telescope/blob/0764c770d57641e2bd866fb80e9beaa3253873d6/test/syntax-highlight.test.js#L8) is the test (using return to test the code)

Reply:
If we return `dom.window.document.body.innerHTML`  like in `syntax-highlight` for this code , the result being passed back would be encoded, which isn't useful as we expect decoded html for the test.